pino.js

```diff
@@ -48,21 +48,27 @@ function defineLevelsProperty (onObject) {
     },
     enumerable: true
   })
-  Object.defineProperty(onObject.levels.values, 'silent', {value: 100})
-  Object.defineProperty(onObject.levels.labels, '100', {value: 'silent'})
+  Object.defineProperty(onObject.levels.values, 'silent', {value: Number.POSITIVE_INFINITY})
+  Object.defineProperty(onObject.levels.labels, Number.POSITIVE_INFINITY.toString(), {value: 'silent'})
```

Comment:
After this change it wouldn't be possible anymore to set the `silent` loglevel by assigning `100`.
Would that be a breaking change?